src/traffic_server/InkAPI.cc

```diff
@@ -9250,6 +9250,7 @@ TSSslContextFindByAddr(struct sockaddr const *addr)
   return ret;
 }
 
+#if 0
```

Comment:
How will this TSAPI be replaced?

Reply:
It would take some effort.  The strings would have to be returned in groups of 4 not 2.  Either we'd have to add an array of lengths, or store the null terminator in the map keys.